### PR TITLE
Added option to remove editors from Archives and Algorithms

### LINKED ISSUE #1195: Remove editors from algorithm editors group
Algorithm editors should be able to remove other editors from the group, but not themselves. Similar to reader studies and challenges.

### PR DESCRIPTION
**Pull request by miriam-groeneveld**
Closes #1195

### CODE REVIEW

File: app/tests/archives_tests/test_forms.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    ArchiveFactory,
+    ArchivePermissionRequestFactory,
+)
+from tests.factories import UserFactory, WorkstationFactory
+from tests.utils import get_view_for_user
+
+
+@pytest.mark.django_db
+def test_editor_update_form(client):
+    archive, _ = ArchiveFactory(), ArchiveFactory()
```
Comment (MikeOverkamp-diag) [inline app/tests/archives_tests/test_forms.py:21]: Why create the second archive here if you're not going to use it?

File: app/tests/archives_tests/test_forms.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    assert response.status_code == 200
+
+    archive.refresh_from_db()
+    assert archive.editors_group.user_set.count() == 1
+    assert not archive.is_editor(user=new_editor)
+
+
+@pytest.mark.django_db
+def test_user_update_form(client):
+    archive, _ = ArchiveFactory(), ArchiveFactory()
```
Comment (MikeOverkamp-diag) [inline app/tests/archives_tests/test_forms.py:63]: Same question :slightly_smiling_face:

**Review by MikeOverkamp-diag — CHANGES_REQUESTED** [review]
LGTM, just one minor thing

File: app/tests/archives_tests/test_forms.py (hunk 2 above)
Comment (miriam-groeneveld) [reply app/tests/archives_tests/test_forms.py:63]: it's a copy artifact ;)

**Review by MikeOverkamp-diag — APPROVED** [review]

**Pull request merged**